operator/builtin/transformer/remove/remove.go

```diff
@@ -47,7 +47,8 @@ func (c RemoveOperatorConfig) Build(context operator.BuildContext) ([]operator.O
 	if err != nil {
 		return nil, err
 	}
-	if c.Field == entry.NewNilField() {
+
+	if c.Field == NewNilRootableField() {
```

Comment:
```suggestion
	if c.Field.Field == entry.NewNilField() {
```

I think this should work, and then we don't need `NewNilRootableField()`